app/Models/Concerns/HasPassword.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Models\Concerns;
+
+use App\Events\User\PasswordChanged;
+use App\Events\User\PasswordRemoved;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Session;
+use Laravel\Passport\Token;
+
+trait HasPassword
+{
+    private $passwordAttributeName = 'password';
```

Comment:
Could be `const`

Reply:
DId this because traits can't have constants :/ Could either define on the user model itself, but I thought it would be nicer for it to be contained in the trait?